### PR TITLE
merging datacard processes

### PR DESCRIPTION
**Pull request by irenedutta23**
Merging several processes in datacards and histograms

### CODE REVIEW

File: tests/hmm/plotting.py [hunk 1]
```diff
@@ -366,15 +366,20 @@ def create_variated_histos(
...
-
 def create_datacard(dict_procs, parameter_name, all_processes, histname, baseline, variations, weight_xs):
     
     ret = Results(OrderedDict())
     event_counts = {}
-
     hists_mc = []
- 
-    #print("create_datacard processes=", all_processes)
+    proc_grps = [
```
Comment (jpata) [inline tests/hmm/plotting.py]: To make it simpler to manage the code in the future, could you load `proc_grps` from pars.py, so it's not hardcoded?

File: tests/hmm/plotting.py [hunk 2]
```diff
@@ -773,11 +827,14 @@ def PrintDatacard(categories, event_counts, filenames, ofname):
 
                 histos = {s: res[s][analysis][var] for s in mc_samples + ["data"]}
                 print(era, analysis, var)
+                combined_signal_samples= ["ggh_amcPS", "vbf", "vh", "tth"]
```
Comment (jpata) [inline tests/hmm/plotting.py]: Same, here, could you load these from pars.py? Just to be able to change and review all the configuration options in one place.

File: tests/hmm/plotting.py (hunk 1 above)
Comment (irenedutta23) [reply tests/hmm/plotting.py]: done

File: tests/hmm/plotting.py (hunk 2 above)
Comment (irenedutta23) [reply tests/hmm/plotting.py]: done

**Comment by jpata** [comment]
Great, I saw you changed where the group information is stored! From your point of view, should I give a try to running this? Or we can go ahead and just merge if you are happy.

**Comment by irenedutta23** [comment]
You can try to run it, there maybe something which I missed. Although I think it should work

**Comment by jpata** [comment]
Okay, I tested and I had no issues. I took a quick look at the datacards and resulting plots so I think all is good. Merging!

**Pull request merged**